### ISSUE TITLE
Customize toolbars positiong and styling

### ISSUE THREAD

**Issue by franleplant** [Leaflet/Leaflet.draw#453]
Hi!
        I'd like to customize the way the toolbars are displayed and the position where they are displayed, is there some lower level API I can base upon to do it?
Thanks!
Fran

**Comment by thomasgwatson** [comment]
Currently it seems like you can only overwrite the CSS....except for the icons, where they are based on a static sprite asset that you can't edit programmatically.

I have been wondering about creating my own control gui elements and associating the right event listeners to them. This is pretty straight forward with controls like zoom in, zoom out... but I think I'd have to deeply delve into leaflet.draw to know what is going on...

**Comment by franleplant** [comment]
You can invoke the draw actions manually: https://github.com/Leaflet/Leaflet.draw/blob/master/dist/leaflet.draw-src.js#L2413

**Comment by thomasgwatson** [comment]
I clarify, via the handlers? (for example, for a rectangle button, on click:
new L.Draw.Rectangle(map, this.options.rectangle))

**Comment by franleplant** [comment]
Exactly, this is a snippet for start drawing a rectangle (note that all events will be fired correctly):

```js
// Create the polygon
let handler = L.Draw.Polygon(map, {})

// Start drawing it
handler.enable()
```

`map` is the map object that Leaflet returns when you create one.


So basically you can trivially attach that snippet to any custom controls.

**Comment by ddproxy** [comment]
This allows you to wire your own toolbar or buttons outside of leaflet. Will add documentation for this.

**Comment by gangstaJS** [comment]
Any progress with the feature?
